### PR TITLE
refactor(document-handling): Hacking session split document modules

### PR DESCRIPTION
**Pull request by mathias-vandaele**
## Description

Hacking session on document handling

## Related issues

closes #

## Checklist

- [ ] PR has a **milestone** or the `no milestone` label.

### CODE REVIEW

File: connector-sdk/feel-wrapper/src/main/java/io/camunda/connector/feel/FeelEngineWrapper.java [hunk 1]
```diff
@@ -59,7 +59,8 @@ public FeelEngineWrapper() {
             .registerModule(DefaultScalaModule$.MODULE$)
             .registerModule(new JavaTimeModule())
             .registerModule(new Jdk8Module())
-            .registerModule(new JacksonModuleDocument(new DocumentFactoryImpl(null), null))
+            .registerModule(
+                new JacksonModuleDocumentDeserializer(new DocumentFactoryImpl(null), null))
```
Comment (johnBgood) [inline connector-sdk/feel-wrapper/src/main/java/io/camunda/connector/feel/FeelEngineWrapper.java:63]: we might need the serializer only righ?

File: connector-sdk/jackson-datatype-document/src/main/java/io/camunda/connector/document/annotation/jackson/JacksonModuleDocumentSerializer.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import io.camunda.connector.document.annotation.jackson.serializer.DocumentSerializer;
+import io.camunda.document.Document;
+
+public class JacksonModuleDocumentSerializer extends SimpleModule {
+
+  public JacksonModuleDocumentSerializer() {}
+
+  @Override
+  public String getModuleName() {
+    return "JacksonModuleDocument";
```
Comment (johnBgood) [inline connector-sdk/jackson-datatype-document/src/main/java/io/camunda/connector/document/annotation/jackson/JacksonModuleDocumentSerializer.java:30]: we need to update the name, same for the other module

File: connector-sdk/jackson-datatype-document/src/main/java/io/camunda/connector/document/annotation/jackson/JacksonModuleDocumentSerializer.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  @Override
+  public Version version() {
+    // TODO: get version from pom.xml
+    return new Version(0, 1, 0, null, "io.camunda", "jackson-datatype-document");
```
Comment (johnBgood) [inline connector-sdk/jackson-datatype-document/src/main/java/io/camunda/connector/document/annotation/jackson/JacksonModuleDocumentSerializer.java:36]: same here

**Review by johnBgood — DISMISSED** [review]
Just some questions, otherwise great

**Comment by sbuettner** [comment]
Looks good, can you exclude all files not relevant to the document module changes?

**Comment by mathias-vandaele** [comment]
> Looks good, can you exclude all files not relevant to the document module changes?

Hey @sbuettner, it was said during the hacking session that we could be doing both same times, but you are right, to be cleaner, let's separate both PR

**Review by sbuettner — APPROVED** [review]

**Pull request merged**